localscripts/generate-po2json.sh

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+[ -x "$(command -v npm)" ] || { >&2 echo "npm is required but not installed, exiting."; exit 1; }
+
+echo ""
+echo "Installing po2json..."
+echo ""
+npm install po2json
+
+echo ""
+echo "Generating .po to .json files..."
+echo ""
+
+pofilenames=$(ls ./translations/planet4-plugin-gutenberg-blocks/languages/*.po)
+for eachpofile in $pofilenames
+do
+   subdomain='planet4-blocks-backend'
+   if [[ "$eachpofile" =~ .*"$subdomain".* ]]; then
+     # Generate json file from .po file. (Note: The o/p filename should be ${domain}-${locale}-${handle}.json)
+     npx po2json "$eachpofile" "${eachpofile/.po/}-planet4-blocks-script.json" -f jed1.x;
```

Comment:
Can we choose this name? I guess `"${eachpofile/.po/}-planet4-blocks-backend.json"` would be more appropriate?

Reply:
Yes, we can. but it needs extra changes in blocks repo.

The json file name should be in `${domain}-${locale}-${handle}.json` format.
In blocks repo, we have `planet4-blocks-script` [handle name](https://github.com/greenpeace/planet4-plugin-gutenberg-blocks/blob/planet-5377-js-translation/classes/class-loader.php#L331)

Reply:
Ok, if the other PR is too many changes for now then we can merge this one with `script` name and change it later :+1: